livesql/binlog.go

```diff
@@ -211,7 +216,7 @@ func parseBinlogRow(table *sqlgen.Table, binlogRow []interface{}, columnMap *col
 		scanner := scanners[i].(*fields.Scanner)
 		scanner.Target(field)
 		if err := scanner.Scan(binlogRow[j]); err != nil {
-			return nil, err
+			return nil, fmt.Errorf("binlog: `%s`.`%s` error: %v", table.Name, table.Columns[i].Name, err)
```

Comment:
if we want things to be quoted, we can use %q and it will sanitize strings in quotes.

Reply:
I was just trying to give a sense of what the tables look like